
Header header
float64 current_time
float64 iterationTime
float64 timeConsumed
